contracts/TokenHolderRevenueFund.sol

```diff
@@ -96,18 +90,31 @@ contract TokenHolderRevenueFund is Ownable, AccrualBeneficiary, Servable, SelfDe
     // Functions
     // -----------------------------------------------------------------------------------------------------------------
     function setRevenueTokenAddress(address newAddress) public onlyOwner notNullAddress(newAddress) {
```

Comment:
Although not changed in this PR I suggest this function signature be changed to `changeRevenueToken(RevenueToken newRevenueToken) public` in order to fit the pattern used in other contracts.